Resources/doc/cookbook/recipe_dynamic_form_modification.rst

```diff
@@ -45,5 +45,8 @@ Then, you should be able to dynamically add needed fields to the form:
             if ($this->isCurrentRoute('create')) {
                 $formMapper->add('name', 'text');
             }
+
+            //$this->isCurrentRoute('create'); // is create page?
```

Comment:
I'm sorry. It's just some examples here, where the code snippet should be placed?